wire-ios-request-strategy/Sources/Payloads/Processing/ConversationEventPayloadProcessor.swift

```diff
@@ -453,13 +459,19 @@ final class ConversationEventPayloadProcessor {
         updateMessageProtocol(from: payload, for: conversation)
 
         if conversation.mlsGroupID != nil {
-            createOrJoinSelfConversation(from: conversation)
+            Task {
```

Comment:
I would expected this method `updateOrCreateSelfConversation()` tor turn async because it called from the async method `processEvents()` in `ConversationEventProcessor`